File: Ejercicio/c sharp/JSON/ConvertirJSON.cs
using System;
using System.Text.Json;

class Persona {
    public string nombre { get; set; }
    public int edad { get; set; }
    public string profesion { get; set; }
}

class Ejemplo3 {
    static void Main() {
        string texto = "{\"nombre\":\"Carlos\",\"edad\":30,\"profesion\":\"Ingeniero\"}";
        Persona p = JsonSerializer.Deserialize<Persona>(texto);
        Console.WriteLine($"Nombre: {p.nombre}, Edad: {p.edad}");
    }
}

File: Ejercicio/GO/JSON/ConvertirJSON.go
package main

import (
	"encoding/json"
	"fmt"
)

type Persona struct {
	Nombre    string `json:"nombre"`
	Edad      int    `json:"edad"`
	Profesion string `json:"profesion"`
}

func main() {
	texto := `{"nombre":"Carlos","edad":30,"profesion":"Ingeniero"}`
	var p Persona
	json.Unmarshal([]byte(texto), &p)
	fmt.Printf("Nombre: %s, Edad: %d\n", p.Nombre, p.Edad)
}
